tests/metrics/classification/test_auc.py

```diff
@@ -0,0 +1,76 @@
+from collections import namedtuple
+from functools import partial
+
+import pytest
+import torch
+import numpy as np
+
+from pytorch_lightning.metrics.classification.auc import AUC
+from pytorch_lightning.metrics.functional.auc import auc
+from sklearn.metrics import auc as _sk_auc
+from tests.metrics.utils import MetricTester, NUM_BATCHES
+
+torch.manual_seed(42)
+
+
+def sk_auc(x, y, reorder=False):
+    x = x.flatten()
+    y = y.flatten()
+    if reorder:
+        idx = np.argsort(x, kind='stable')
+        x = x[idx]
+        y = y[idx]
+        print('sk', x, y, idx)
```

Comment:
I think it's better to erase debug print
```python
print('sk', x, y, idx)
```

Reply:
you are right, stupid leftover from debugging :]